docs/config_entries_config_flow_handler.md

```diff
@@ -49,12 +49,13 @@ There are a few step names reserved for system use:
 
 | Step name   | Description                                                                                                                                                   |
 | ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `discovery` | _DEPRECATED_ Invoked if your integration has been discovered by the discovery integration.             |
+| `discovery` | Invoked if your integration has been discovered and the matching step has not been defined.             |
```

Comment:
Implementing this step directly in new integrations is deprecated.

Reply:
😕 The current description is still wrong?
Should this not be marked deprecated in the core docstring?
https://github.com/home-assistant/core/blob/e1a3ef3d6988bd2d31e7cabe5ef5cbd131238089/homeassistant/config_entries.py#L1237-L1242

Reply:
We could do that.